stac_geoparquet/stac_geoparquet.py

```diff
@@ -11,8 +11,14 @@
 import numpy as np
 import shapely.geometry
 
+from urllib.parse import urlparse
+
 from stac_geoparquet.utils import fix_empty_multipolygon
 
+STAC_ITEM_TYPES = ["application/json", "application/geo+json"]
+
+SELF_LINK_COLUMN = "self_link"
```

Comment:
Thoughts on `self_link` vs. `self_href`, or maybe `self_link_href`?

In STAC terms, I tend to think of "link" as specifying the `href`, `rel`, and maybe the `type` and `title`, while the `href` is just the actual URL.

Reply:
I'm trying to think about people that are not so much into the tech.

link > url > href - href is so specific that some data scientists don't even know what that is. ;-)